.github/workflows/publish.yml

```diff
@@ -3,53 +3,52 @@ name: Deploy site
 on:
```

Comment:
(Unrelated to this file)
Do you think it's worth having a CI build check linting, so after you fixing stuff it stays consistent?

Reply:
I don't know of any particular one; would be nice to have though.